sample/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
You can start by specifying a port of your choice. Use yaml instead of properties